launcher/aboutProject/aboutproject_moc.cpp

```diff
@@ -16,6 +16,20 @@
 #include "../../lib/GameConstants.h"
 #include "../../lib/VCMIDirs.h"
 
+void AboutProjectView::hideAndStretchWidget(QGridLayout * layout, QWidget * toHide, QWidget * toStretch)
+{
+	toHide->hide();
+
+	int index = layout->indexOf(toStretch);
+	int row;
+	int col;
+	int unused;
+	layout->getItemPosition(index, &row, &col, &unused, &unused);
+	layout->removeWidget(toHide);
+	layout->removeWidget(toStretch);
+	layout->addWidget(toStretch, row, col, 1, -1);
```

Comment:
doesn't it stretch automatically if you simply remove & delete button from the row?

Reply:
If entire column is empty Qt will resize it to ~0px
But this column in layout still has one element - "Check for updates" button.

Not sure how to deal with that other than splitting layouts. And splitting layouts would make columns in "Build information" and "data directories have different width which does not looks nice.

Reply:
Yeah, the only way would be to replace grid with rows